### PR TITLE
:sparkles: Translate command

### PR DESCRIPTION
**Pull request by Gilbert142**
**Please describe the changes this PR makes and why it should be merged:**

**What does this PR do:**
- [ ] This PR changes internal functions, modules and/or utilities
  - [ ] This PR modifies the Extension API
- [x] This PR modifies commands
  - [ ] This PR changes metadata for commands (such as usage), updated in `commands.js`
- [ ] This PR modifies Web processing and/or content
  - [ ] This PR modifies static/ejs content
  - [ ] This PR modifies request processing (controllers & middleware)
  - [ ] This PR modifies paths to existing resources (routes)
- [ ] This PR **only** includes non-code changes, like changes to default configurations, README, typos, etc.

### CODE REVIEW

File: Commands/Public/translate.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			footer: {
+				text: `Translated using Microsoft Translator. The translated text might not be 100% accurate!`,
+			},
+		},
+	});
+	const onFail = (err, src) => {
+		msg.send({
+			embed: {
+				color: Colors.SOFT_ERR,
+				description: `Something went wrong while trying to ${src === null ? "detect your text's langauge" : `translate your ${src} text`}! 😵`,
```
Comment (vladfrangu) [inline Commands/Public/translate.js]: langauge

File: Commands/Public/translate.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+			},
+		});
+		if (err) winston.debug(`Failed to ${src === null ? "auto-detect language" : "translate text"} for ${commandData.name} command.`, { svrid: msg.guild.id, chid: msg.channel.id, msgid: msg.id, err });
+	};
+
+	if (source === "?") {
+		mstranslate.detect({ text }, (err, res) => {
```
Comment (vladfrangu) [inline Commands/Public/translate.js:52]: Non async api?!?!?!?! 🙀

File: Commands/Public/translate.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+			if (err || !res || res === "") {
+				onFail(err, null);
+			} else {
+				mstranslate.translate({ from: res, to: target, text }, (err2, res2) => {
```
Comment (vladfrangu) [inline Commands/Public/translate.js]: CALLBACK HELL?!?!

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: Commands/Public/translate.js (hunk 3 above)
Comment (Gilbert142) [reply Commands/Public/translate.js]: It's 2 callbacks. **2**

...

But fine I'll fix it

File: Commands/Public/translate.js (hunk 2 above)
Comment (Gilbert142) [reply Commands/Public/translate.js:52]: I've gone bad 😎

File: Commands/Public/translate.js (hunk 1 above)
Comment (Gilbert142) [reply Commands/Public/translate.js]: shtu up

File: Commands/Public/translate.js (hunk 2 above)
Comment (vladfrangu) [reply Commands/Public/translate.js:52]: You've gone mad 🙄

Comment (wasdennnoch) [reply Commands/Public/translate.js:52]: Wrap it in your own promise :P

Comment (Gilbert142) [reply Commands/Public/translate.js:52]: Do it your own damn self 😎

Comment (vladfrangu) [reply Commands/Public/translate.js:52]: *tsk tsk* Thats some sassy sass right there

**Pull request merged**